lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -569,7 +581,11 @@ defmodule ExUnit.Case do
       end
 
       def __ex_unit__(:config) do
-        {unquote(async?), unquote(Macro.escape(parameterize))}
+        %{
+          async?: unquote(async?),
+          async_partition_key: unquote(async_partition_key),
```

Comment:
We need to Macro.escape the key as well, someone could have a group of `{1, 2, 3}`.

Reply:
https://github.com/elixir-lang/elixir/pull/13897/Done in commits/07e5429ff03d3c51ff0db7a1931abdf2d99f3c10